easybuild/easyconfigs/g/GPAW/GPAW-1.2.0-customize.patch

```diff
@@ -0,0 +1,36 @@
+--- gpaw-1.2.0-orig/customize.py	2017-05-24 11:57:38.765096571 +0200
```

Comment:
@boegel thanks for helping, added description, author and institution (see ae2977a88cb80bb388f17852e37ff11838037655)